### ISSUE TITLE
Production deployment not working, the content is not being put on the DNS. Instead still seeing cpanel's file manager placeholder content

### ISSUE THREAD

**Issue by JulsRicketti** [vercel/vercel#3098]
Not sure if this is the right spot to ask this question, so my apologies if it is not, but I don't know where else would be more appropriate.

I have a simple app that is ready to go to production. The domain name has already been purchased previously and everything, so now all I had to do was the deployment of the 
 webapp to production.

Following the docs on the site, all was set up on the dashboard. I also got an email that requested I change my current name servers to the ZEIT servers. I did this as requested and saw the proper DNS on the dashboard as expected.

However, if I actually try to go to the production app, I get whatever nonsense placeholder content was on cpanel's file manager. I have no clue what to do from here. Did I do something wrong during the set up? Or miss something? Is this a problem with the purchased domain name itself?

Thanks!

**Comment by thasophearak** [comment]
@JulsRicketti Thank for the issue.

I presumed that you followed this https://zeit.co/docs/v2/custom-domains/#adding-a-domain

If that the case, can you visit your project's domain tab, see if your domain is verified? It should like something similar below screenshot.
![image](https://user-images.githubusercontent.com/5785105/65852317-feff3280-e37f-11e9-9b7f-d650bb5ed1d0.png)

**Comment by JulsRicketti** [comment]
Thanks for the help @sophearak!

Yes, my domain is verified here is the screenshot of what that section looks like now:

![Screen Shot 2019-09-30 at 9 01 21 AM](https://user-images.githubusercontent.com/886808/65881154-f9debb00-e360-11e9-8dab-3ac248e12927.png)

**Comment by JulsRicketti** [comment]
Holly crap! You won't believe this! (or maybe you will... The world of deployment is a funny thing XD)
I decided to check on the site right after I posted this answer. And now it's up! It had been stuck with the previous problem since Saturday afternoon, was this perhaps an issue that I simply needed to wait it out? Was it something being cached?

Before closing the issue, is there anything that could have caused it?

Thank you so much for your time!
